add grant testing faqs (#7415)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/faq/testing-faq/index.md

-   **PA on-device scoring data**: DSP to SSP
    -   Bid: How to handle multi-currencies. Each SSP may have different requirements.
    -   Ad object (returned from `generateBid()`): The SSP may use this for scoring and publisher controls. DSPs and SSPs need to align on what data is included and on a data structure.
-   **PA win reporting**: SSP to DSP
    -   What SSP's `reportResult()` will make available to the DSP's `reportWin()`.
-   **Post-auction reporting**: DSP to SSP. Context: The DSP must capture and delegate post-auction reports to the SSP. If the SSP needs clicks, views, viewability metrics, the DSP must enable them to receive these events.
    -   What reports the SSP will need.
    -   The methodology for measuring that event (such as, viewability definition).
    -   DSP implementation of the `reportEvent()` call to support the SSP requirements.
    -   SSP implementation of the `registerAdBeacon()` lining up the event naming with what the DSP will trigger in the creative.

**Are the Attribution Reporting API and the Aggregation Service ready and available for testing?**

> The Attribution Reporting API will be generally available and ramp-up is already in progress. With that, Attribution Reporting will soon be available for use on 100% of traffic. Note that Attribution Reporting can be used with all ads, not just those served by Protected Audience.

Reviewer: I feel like if a question asks about the Aggregation Service, the response should at least mention it once, even if the assumption is people understand the context.

Author: Good catch.

Author: Asking now

Author: still waiting. Might get answer today.

**Are there geographic requirements for testing?**

There are no geographic requirements. It's up to each tester to determine geographic considerations as part of their test.

**How does the CMA guidance align with Chrome-facilitated testing?**

Chrome facilitated testing modes are aligned with the CMA's guidance for quantitative testing of Privacy Sandbox. In the CMA's guidance the treatment group relies on Privacy Sandbox technologies without third-party cookies. Control group 1 uses third-party cookies and not Privacy Sandbox, and Control group 2 uses neither Privacy Sandbox nor third-party cookies. With general availability, the Privacy Sandbox technologies will be available on all Chrome traffic, and the ad tech can choose to use Privacy Sandbox technologies in certain population groups and not in others. Additionally, ad techs can leverage the Mode A Chrome-facilitated testing traffic to coordinate these population groups across multiple parties. 

> Starting in Q1 2024, Chrome will deprecate third-party cookies on 1% of traffic and this is referred to as the Mode B Chrome-facilitated testing traffic in Chrome developer documentation. In a small fraction of Mode B Chrome-facilitated testing traffic, Privacy Sandbox technologies will also not be available in addition to third-party cookies being deprecated. Using a combination of all of the above Chrome-facilitated testing modes, ad techs will be able to align their test setups to the testing guidance published by the CMA. (Refer to [Chrome-facilitated testing](/docs/privacy-sandbox/chrome-testing/#:~:text=Mode%20A%3A%20Ad%20techs%20can,portion%20of%20all%20Chrome%20users.) which defines Mode A and Mode B and [CMA Guidance](https://assets.publishing.service.gov.uk/media/649d6a5f45b6a2000c3d455f/20230629_CMA_industry_testing_update_B.pdf) defines Control 1, Control 2, Treatment groups (in point number 11)

Reviewer: Formatting of the last sentence is a bit off—a bracket opened and not closed, a bracket within a bracket, no full-stop at the end et cetera

**Is there a contact at the CMA we could connect with to better understand what is required for the submitted final report?**

> You can email the CMA case team at <a href="mailto:privacysandbox@cma.gov.uk">privacysandbox@cma.gov.uk</a> and read more about their testing proposal guidance [here](https://assets.publishing.service.gov.uk/media/649d6a5f45b6a2000c3d455f/20230629_CMA_industry_testing_update_B.pdf). (For additional reference, the CMA lists all their contacts at the bottom of their [Privacy Sandbox page](https://www.gov.uk/cma-cases/investigation-into-googles-privacy-sandbox-browser-changes))

Reviewer: Remove link from [here] and instead add descriptive link text. This sentence too has unnecessary brackets imo

> You can email the CMA case team at <a href="mailto:privacysandbox@cma.gov.uk">privacysandbox@cma.gov.uk</a> and read more about their testing proposal guidance [here](https://assets.publishing.service.gov.uk/media/649d6a5f45b6a2000c3d455f/20230629_CMA_industry_testing_update_B.pdf). (For additional reference, the CMA lists all their contacts at the bottom of their [Privacy Sandbox page](https://www.gov.uk/cma-cases/investigation-into-googles-privacy-sandbox-browser-changes))

Reviewer: ```suggestion
You can email the CMA case team at <a href="mailto:privacysandbox@cma.gov.uk">privacysandbox@cma.gov.uk</a> and read more about their testing proposal guidance [here](https://assets.publishing.service.gov.uk/media/649d6a5f45b6a2000c3d455f/20230629_CMA_industry_testing_update_B.pdf). (For additional reference, the CMA lists all their contacts at the bottom of their [Privacy Sandbox page](https://www.gov.uk/cma-cases/investigation-into-googles-privacy-sandbox-browser-changes)).
```

> **Could you please clarify quantitative testing success metrics?**

Reviewer: ```suggestion
**Can you clarify the quantitative testing success metrics?**
```

Grant participants can define and share as many additional KPIs as they want, beyond those specified in the SOW. The minimum test results that need to be shared are listed in the [CMA Guidance](https://assets.publishing.service.gov.uk/media/649d6a5f45b6a2000c3d455f/20230629_CMA_industry_testing_update_B.pdf). This guidance is purposefully "light" and broad so the CMA can compare many results from different ad techs.

> **Is there a volume threshold for testing? (Control group ratios vs. Modes ratios, etc.) Do you have additional guidance on valuable KPIs to test beyond the CMA's testing guidance?**

Reviewer: ```suggestion
**Is there a volume threshold for testing, for example, control group ratios versus modes ratios? Do you have additional guidance on valuable KPIs to test beyond the CMA's testing guidance?**
```

> You can draft these volume/scale requirements in your Integration Plan; we will work with you to align on the scale required given your business model. You can include specific metrics and tests in your Testing Plan. Please keep in mind that the Testing Plan should take into account Mode A and Mode B of Chrome-facilitated testing. We can provide feedback on these once you submit them.

Reviewer: ```suggestion
You can draft these volume / scale requirements in your integration plan. We will work with you to align on the scale required given your business model. You can include specific metrics and tests in your testing plan. Please keep in mind that the testing plan should take into account Mode A and Mode B of Chrome-facilitated testing. We can provide feedback on these once you submit them.
```
